heart/test/ionicmodels/TestCodegen.hpp

```diff
@@ -159,8 +164,13 @@ class TestCodegen : public CxxTest::TestSuite
         double end_time = 1000.0; //One second in milliseconds
         double i_ionic_end_time = 60.0; // ms
         double i_ionic = 1.9411; // test value
+        double sampling_interval = 1.0; // for testing codegen error check
 
         // Normal model
+        // test error from codegen about concentration
```

Comment:
Might want to make these TS_ASSERT_THROWS_CONTAINS("Concentration JSR_calcium_concentration below 0") as they probably won't always agree to 6 decimal places.